File: official/docs/node/v6/rates/regenerate.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient(process.env.EASYPOST_API_KEY);

(async () => {
  const shipment = await client.Shipment.retrieve('shp_...');

  const rates = await client.Shipment.regenerateRates(shipment.id);

  console.log(rates);
})();

File: official/docs/python/v8/rates/regenerate.py
import easypost

client = easypost.EasyPostClient("EASYPOST_API_KEY")

shipment = client.shipment.retrieve("shp_...")

shipment_with_regenerated_rates = client.shipment.regenerate_rates(shipment.id)

print(shipment_with_regenerated_rates)
